.github/workflows/support-window.yml

```diff
@@ -0,0 +1,54 @@
+name: Update support window diagram
+on:
+  # When translations are added/removed
+  push:
+    paths: README*.md
+  # Manually, when TypeScript is released
+  # https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          personal <- Sys.getenv("R_LIBS_USER")
+          dir.create(personal, recursive = TRUE)
+          install.packages(c("ggplot2", "svglite"), personal)
+        shell: Rscript {0}
+      - name: Fetch TypeScript versions and release dates from npm
+        run: |
+          npm view --json typescript time |
+            jq '
+              to_entries
+              # Earliest non-pre-release <major>.<minor>
+              | map(
+                (.key |= capture("^(?<version>[0-9]+\\.[0-9]+)\\.[0-9]+$").version)
+                | select(.key)
+              )
+              | unique_by(.key)
+              | from_entries
+            ' > docs/support-window.json
+      - run: git rm --ignore-unmatch 'docs/support-window*.svg'
+      - name: Make SVG diagram
+        run: |
+          export LOCPATH=.locale
+          mkdir "$LOCPATH"
+          for readme in README*.md; do (
+            if [ "$readme" != README.md ]; then
+              LANG=${readme:7:-3}
+              # Use e.g. en_EN if it exists, or the first ${LANG}_*
+              # otherwise
+              locale=/usr/share/i18n/locales/${LANG}_${LANG^^}
+              [ -e "$locale" ] || locale=(/usr/share/i18n/locales/${LANG}_*)
+              localedef --charmap UTF-8 --inputfile "$locale" "$LOCPATH/$LANG"
+            fi
+            jq --raw-output 'to_entries[] | map(values) | @tsv' docs/support-window.json |
+              Rscript scripts/support-window.R
+          ) done
+      - run: git add docs/support-window.*
+      - run: git config user.name .github/workflows/support-window.yml
+      - run: git config user.email bot@typescriptlang.org
+      - run: git diff --quiet --cached || git commit --message '[README] 🤖 Update support window diagram'
```

Comment:
Double quotes

Reply:
I have a habit of no-interpolation by default, and explicitly turning it on vs. explicitly turning it off. The Google style guide [doesn't recommend one or the other](https://google.github.io/styleguide/shellguide#quoting) (not that it governs this project, it's just the shell style guide I know of). Just curious about the reason for defaulting to double quotes?

Reply:
My main reason is that `"`s is generally more common in scripts that I see, which makes them more understandable.

[There are other reasons that I prefer to avoid `'`s: they tend to be more surprising (especially when people try to `\`-escape stuff and fail), and it's kind of surprising when `$`s don't work (for people who don't know that the two quotes are very different).  But those are actually kind of encapsulated by the above reason.  Yet another thing to consider is that people very often jump onto here-docs/strings in cases where the same could be done with `'`s.]

Reply:
:heavy_check_mark: Done.